### PR TITLE
Added suggested commands to JShell sample

### PR DESCRIPTION
**Pull request by sjmaple**
As requested, here's the commands we have for JShell.

### CODE REVIEW

File: session-3-jshell/java9-improvements-to-try.txt [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import java.time.*;
+void printf(Object a) { System.out.println(a); }
+
+* Collections of factory methods: 
+1-10 argumentes + varargs: implementation classes 2 elements vs N elements.
+    * demo: 
+        * List.of(1, 2, 3, 4, 5)
+        * Map.of("hello", "world", "status", "mad")
+        * Set.of(1,2,4)
+        * Set.of(1,2,4)
```
Comment (neomatrix369) [inline session-3-jshell/java9-improvements-to-try.txt]: Is this repeat deliberate?

**Comment by neomatrix369** [comment]
@sjmaple Any chance of adding this fantastic list to the already existing - https://github.com/AdoptOpenJDK/jdk9-jigsaw/blob/master/session-3-jshell/README.md

**Comment by sjmaple** [comment]
Yeh, I think ordering of the Set isn't consistent... maybe :)

Thanks -- Simon

**Comment by neomatrix369** [comment]
@sjmaple I'll merge this now and merge the two lists together to make one resource for JShell. Thanks plenty for sharing it with us.

**Pull request merged**